generators/client/templates/angular/webpack/webpack.custom.js.ejs

```diff
@@ -84,6 +84,37 @@ module.exports = (config, options) => {
         })
       );
     }
+
+    <%_ if (cypressCoverage) { _%>
+    if (process.env.INSTRUMENT) {
+      // For cypress coverage, we DO NOT want the browser to automatically synchronize
+      config.plugins.forEach(plugin => {
+        if (plugin.browserSyncOptions) {
+          plugin.browserSyncOptions.ghostMode = {
+            clicks: false,
+            location: false,
+            forms: false,
+            scroll: false,
+          };
+          plugin.browserSyncOptions.open = false;
+        }
+      });
+      // We also add istanbul loader
+      config.module.rules.push({
+        test: /\.(js|ts)$/,
+        use: [
+          {
+            loader: 'istanbul-instrumenter-loader',
+            options: { esModules: true },
+          },
```

Comment:
I just saw that istanbul-instrumenter-loader was archived and conflicts with webpack 5 https://github.com/webpack-contrib/istanbul-instrumenter-loader/issues/110, so no angular 12, react or vue.

This is good for a blueprint, not for inclusion at jhipster.
Can you switch to babel-loader with istanbul plugin?

```suggestion
          {
            loader: 'babel-loader',
            options: {
              plugins: ['istanbul'],
            },
          },
```